Document/0x05b-Basic-Security_Testing.md

```diff
@@ -2144,6 +2144,9 @@ For information on disabling SSL Pinning both statically and dynamically, refer
 - Magisk Modules repository - <https://github.com/Magisk-Modules-Repo>
 - Magisk Trust User Certs module - <https://github.com/NVISO-BE/MagiskTrustUserCerts/releases>
 - Mitm-relay - <https://github.com/jrmdev/mitm_relay>
+- Mitm Proxy - <https://mitmproxy.org>
+- Upstream Proxy - <https://github.com/inaz2/proxy2>
+- JWT Secret Key Cracker - <https://github.com/brendan-rius/c-jwt-cracker>
```

Comment:
This is the references section. Unfortunately we cannot add references to tools without actually referencing them in the chapter. If you think that some tools are missing or worth including please feel free to do so, maybe another chapter is more appropriate:
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05g-Testing-Network-Communication.md